src/main.go

```diff
@@ -58,10 +58,18 @@ func main() {
 	// Can change folder directory to any folder inside of src/static_data
 
 	wg.Add(1)
-	go func() {
-		parse.ParseRisLiveData(msgChannel, configStruct)
-		wg.Done()
-	}()
+
```

Comment:
[https://github.com/tyakovenko/Fastly-MQP23/pull/17/files#diff-9e185f29fa355d7dd8fdd9c9ff1d0723b85206aa7d37c4eec93997005dc291ebR38](https://github.com/tyakovenko/Fastly-MQP23/pull/17/files#diff-9e185f29fa355d7dd8fdd9c9ff1d0723b85206aa7d37c4eec93997005dc291ebR38)

Hmm - curious how this is working.